pandas/tests/plotting/frame/test_frame.py

```diff
@@ -2589,6 +2589,14 @@ def test_plot_period_index_makes_no_right_shift(self, freq):
         result = ax.get_lines()[0].get_xdata()
         assert all(str(result[i]) == str(expected[i]) for i in range(4))
 
+    def test_plot_display_xlabel_and_xticks(self):
+        # GH#44050
+        df = DataFrame(np.random.default_rng(2).random((1000, 2)), columns=["a", "b"])
```

Comment:
```suggestion
        df = DataFrame(np.random.default_rng(2).random((10, 2)), columns=["a", "b"])
```

If the test is not dependent on the size of the data, best to keep it minimal

Reply:
Ah makes sense, I've updated it.